### PR TITLE
Feature/visual bubble

### LINKED ISSUE #7: [Feature Request] Visual Bubble
# Overview

Describe the motivation behind your desired functionality.
In order to add some visual distinction between visuals on a dashboard, we are going to encapsulate each graph into a 'bubble'

How would you like the user experience for this feature to work?
Minimal ux build out for component, functionality added on dashboard pages later

# Proposal

Describe what changes need to be implemented to resolve this issue. Be as
detailed as possible so maintainers can best determine how to implement this
new feature.
React component needs to be built out to accept props for datasets and titles to make adding graphs to the dashboard easier later. Should accept 3 different datasets and, title, axis labels, and the desired type of graph.

## Supplementary Information

To give your feature more chance at being implemented, feel free to attach any
other documentation. For something highly-user facing, a UI mock-up or some
other form of wireframe is required.

### PR DESCRIPTION
**Pull request by eboysen**
## Overview

Closes #7 

This branch made major changes to the overall application. Includes all of the graph components as well as a new page component for a dashboard. Graph components for horizontal, vertical, and radial bar charts as well as radar charts and line graphs were added. A grid based, dashboard was created to scale with devices and allow graphs to easily scale to fit the screen.

## What Changed
The Components folder now holds a folder of graph components. Most of these take a very similar form as they all are implementations of apex charts. The new dashboard page is accessible through the /dashboard route. 

### IMPORTANT NOTE
There are a couple of features on this branch that are entirely placeholders for development. These include the Top Menu and Search Grid Components. These were mostly left in place in order to allow for developers to get a feel for what the overall layout of the dashboard page would be, not as final implementations.

Side Note
The Radial Bar Chart inherintly does not scale well as it is a circle, it was decided best to optimize it for 1080 displays per Nebula leadership decisions in the past, so it is set to fit height, this means that on mobile displays, the width overflows. This may be adjusted or narrowed in the future, but is an acceptable outcome for the time being.

### CODE REVIEW

File: components/navigation/searchGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <Input
+          placeholder="+ Add Courses"
+          className="w-full text-light"
+        ></Input>
+      </div>
+    </div>
+  );
+};
+
+export default SearchGrid;
```
Comment (iamwood) [inline components/navigation/searchGrid.tsx:28]: This is just to confirm, do we want there to be three boxes there at all times, just with either a search bar or the search term there, or do we want a single box for the autocomplete and add another box for each term added?

Comment (eboysen) [reply components/navigation/searchGrid.tsx:28]: If you check the notes that I have at the top of the PR I made reference to the fact that the Search Grid Component in this branch was a placeholder, in all likelihood this file is gonna be deleted once we move the component you've been working on into the dashboard, so I wouldn't worry about this

File: components/navigation/topMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        </div>
+      </div>
+      <div className="h-full float-right flex align-middle place-items-center justify-center w-10">
+        <HandymanIcon className="w-full " color="inherit" />
+      </div>
+    </div>
+  );
+};
+
+export default TopMenu;
```
Comment (iamwood) [inline components/navigation/topMenu.tsx:23]: Does this need doc comments? It is pretty obvious what it is from the code itself.

Comment (eboysen) [reply components/navigation/topMenu.tsx:23]: Once again I would direct you to my comment in the description of the PR in which I say that this is a placeholder, Chen is working on the actual component that will be implemented eventually, so this placeholder probably does not need to be documented in any substantial manner

**Review by iamwood — APPROVED** [review]
Everything looks fine, and I don't see any problems moving forward.

**Pull request merged**